src/main/java/net/codejava/MvcController.java

```diff
@@ -127,7 +127,11 @@ public String submitDeposit(@ModelAttribute("user") User user) {
 
     if (userPasswordAttempt.equals(userPassword)) {
       // Execute SQL Update command that increments user's Balance by given amount from the deposit form.
+      // Return to welcome screen if amount is negative 
       String balanceIncreaseSql = String.format("UPDATE Customers SET Balance = Balance + %d WHERE CustomerID='%s';", user.getAmountToDeposit(), userID);
+      if (user.getAmountToDeposit() < 0){

```

Comment:
nitpick: I would recommend putting this if-statement before the `balanceIncreaseSql` String is initialized. Just saves a bit of compute time (JVM doesn't waste time making a String object) that might be significant at large scale.